scalafmt-sysops/native/src/main/scala-2.12/org/scalafmt/CompatCollections.scala

```diff
@@ -0,0 +1,12 @@
+package org.scalafmt
+
+object CompatCollections {
+  val JavaConverters = scala.collection.JavaConverters
+
+  // Scala Native does not include ParallelConverters
+  object ParConverters {
+    implicit class XIterable[T](val col: Iterable[T]) extends AnyVal {
+      def compatPar = col
```

Comment:
So back in the previous iteration 3 years ago SN did not have multithreading yet, but even then the single threaded scala native version was usually faster (I imagine this would change at scale). However this comment made me retest performance and I also realized that parallel collections were built into stdlib in scala 2.12, and indeed, they work on Scala Native now. So now, after that change, that would make scala 2.12 on Scala Native the best. Example test:
```
vl-d-1450:main jchyb$ time java --class-path scalafmt.jar org.scalafmt.cli.Cli
Reformatting...
  100,0% [##########] 700 source files formatted

real    0m2,787s
user    0m17,050s
sys     0m0,961s
vl-d-1450:main jchyb$ time ./scalafmt-cli2
Reformatting...
  100.0% [##########] 700 source files formatted

real    0m1,024s
user    0m2,016s
sys     0m0,365s
vl-d-1450:main jchyb$ time ./scalafmt-cli3
Reformatting...
  100.0% [##########] 700 source files formatted

real    0m2,056s
user    0m1,611s
sys     0m0,207s
```
So that makes even the Scala Native version without parallelization better (again, I imagine this would change with a bigger project). Since the 2.12 built-in version works, I can't imagine porting scala-parallel-collections being difficult, so once that is done, the CollectionCompat implementations could be replaced and merged (I added a related comment).